### PR TITLE
add domain association for ios

### PR DESCRIPTION
**Pull request by futurepaul**
this involves a cert update (because there's a new "permission") so hopefully our fastlane config holds up...

https://capacitorjs.com/docs/guides/deep-links

### CODE REVIEW

**Comment by futurepaul** [comment]
update: the fastlane did NOT hold up, but now I learned a valuable lesson about how to update certs with `fastlane match`

File: ios/App/fastlane/README.md [hunk 1]
```diff
@@ -23,6 +23,14 @@ For _fastlane_ installation instructions, see [Installing _fastlane_](https://do
...
 Push a new beta build to TestFlight
 
+### ios build
+
+```sh
+[bundle exec] fastlane ios build
+```
+
+
+
```
Comment (benthecarman) [inline ios/App/fastlane/README.md:33]: nit a bunch of extra spacing here

**Comment by benthecarman** [comment]
@futurepaul you're the only one that can test this. What is the status here? Besides minor formatting nit, lgtm

File: ios/App/fastlane/README.md (hunk 1 above)
Comment (futurepaul) [reply ios/App/fastlane/README.md:33]: that readme is autogenerated by them

**Comment by futurepaul** [comment]
I can't test it until it's merged basically because the `.well-known` stuff needs to be deployed. hard part was getting the new certs for the new entitlement but that's sorted now

I think it's a yolo situation? idk a better way to test

**Comment by benthecarman** [comment]
I think we had to do that with the android ones too. LGTM then

**Review by benthecarman — APPROVED** [review]

**Comment by TonyGiorgio** [comment]
> I think we had to do that with the android ones too. LGTM then

yeah exactly

**Review by TonyGiorgio — APPROVED** [review]

**Pull request merged**